services-core/src/solution_submission/gas_price_increase.rs

```diff
@@ -0,0 +1,77 @@
+use futures::stream::{Stream, StreamExt};
+
+// openethereum requires that the gas price of the resubmitted transaction has increased by at
+// least 12.5%.
+const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.125 * (1.0 + f64::EPSILON);
+
+pub fn minimum_increase(gas_price: f64) -> f64 {
+    (gas_price * MIN_GAS_PRICE_INCREASE_FACTOR).ceil()
```

Comment:
The `ceil` has moved here which also forced me to change the test to use numbers >= 10.